common/src/main/resources/wynntils.mixins.json

```diff
@@ -37,6 +37,11 @@
     "accessors.OptionsAccessor"
   ],
   "compatibilityLevel": "JAVA_17",
+  "injected_interfaces": {
+    "net/minecraft/class_437": [
```

Comment:
Do we need to use the obfuscated name here? That's a bit sad. :-( Can you at least add a comment on what class this really is?

Reply:
(This is likely to break on Minecraft updates...)

Reply:
Yeah this is what loom needs

Reply:
We are at the stage where a minecraft update would already be a boatload of work, so this is not that big of a concern.

Reply:
Actually, I don't think it will. I have been tempted from time to time to try and make a 1.19.2 branch, just to see how much we'd need to change.